examples/energy-management-app/esp32/main/main.cpp

```diff
@@ -146,6 +149,13 @@ static void InitServer(intptr_t context)
 
     // Application code should always be initialised after the initialisation of server.
     ApplicationInit();
+
+#if CONFIG_ENABLE_SNTP_TIME_SYNC
+    char ntpServerUrl[]              = "pool.ntp.org";
+    uint16_t mSyncNtpTimeIntervalDay = 1;
+    chip::Esp32TimeSync mEsp32TimeSync(ntpServerUrl, mSyncNtpTimeIntervalDay);
+    mEsp32TimeSync.Init();
```

Comment:
nit: Would feel more natural if the init takes the arguments rather than the constructor.

Reply:
Init returns a CHIP_ERROR. you should handle it.

Reply:
Changed return type os Init to void